packages/playground/blueprints/src/lib/steps/activate-plugin.ts

```diff
@@ -14,22 +14,23 @@ export interface ActivatePluginStep {
  */
 export const activatePlugin: StepHandler<ActivatePluginStep> = async (
 	playground,
-	{ plugin },
+	{ pluginPath },
 	progress
 ) => {
-	progress?.tracker.setCaption(`Activating ${plugin}`);
-	const pluginsPage = asDOM(
-		await playground.request({
-			url: '/wp-admin/plugins.php',
-		})
+	progress?.tracker.setCaption(`Activating ${pluginPath}`);
+	const requiredFiles = [
+		`${playground.documentRoot}/wp-load.php`,
+		`${playground.documentRoot}/wp-admin/includes/plugin.php`,
+	];
+	const requiredFilesExist = requiredFiles.every((file) =>
+		playground.fileExists(file)
 	);
-
-	const link = pluginsPage.querySelector(
-		`tr[data-slug="${plugin}"] a`
-	)! as HTMLAnchorElement;
-	const href = link.attributes.getNamedItem('href')!.value;
-
-	await playground.request({
-		url: '/wp-admin/' + href,
-	});
+	if (requiredFilesExist) {
```

Comment:
If the files don't exist... what should happen?

Reply:
Throw an error

Reply:
Errors added: https://github.com/WordPress/wordpress-playground/pull/351/commits/d161ef447ee58cebae78500bc0174e3ff57bf055